src/DistributedComputations/distributed_architectures.jl

```diff
@@ -39,7 +39,7 @@ if supplied as positional arguments `x` will be the first argument,
     `Equal()`: divide the domain in `x` equally among the remaining processes (not supported for multiple directions)
     `Fractional(ϵ1, ϵ2, ..., ϵN):` divide the domain unequally among `N` processes. The total work is `W = sum(ϵi)`, 
                                    and each process is then allocated `ϵi / W` of the domain.
-    `Sizes(ϵ1, ϵ2, ..., ϵN)`: divide the domain unequally. EThe total work is `W = sum(ϵi)`, 
+    `Sizes(ϵ1, ϵ2, ..., ϵN)`: divide the domain unequally. The total work is `W = sum(ϵi)`, 
```

Comment:
```suggestion
    `Sizes(n1, n2, ..., nN)`: divide the direction by number of grid points, where `ni` is the number of grid points allocated to process `i`. The total size of the direction is `N = sum(ni)`, 
```

Maybe a different letter than `N` for the number of processors, maybe `P` is better?